src/runtime_src/core/tools/common/tests/TestVerify.cpp

```diff
@@ -16,7 +17,15 @@ TestVerify::TestVerify()
 boost::property_tree::ptree
 TestVerify::run(std::shared_ptr<xrt_core::device> dev)
 {
-  boost::property_tree::ptree ptree = get_test_header();
-  runTestCase(dev, "22_verify.py", ptree);
+  auto device_name = xrt_core::device_query_default<xrt_core::query::rom_vbnv>(dev, "");
+  boost::property_tree::ptree ptree;
+  if(device_name.find("IPU") != std::string::npos) {
```

Comment:
You should definitely comment that this is a temporary hack! At least I hope it is going to be... 
The idea makes sense to me though. I do see a lot of problems with scaling this solution, especially as we move into multiple device types that would all use the IPU test.

Reply:
I added a to-do tag. We can tackle this as a part of tests refactoring story